releases/README.md

```diff
@@ -0,0 +1,104 @@
+# Release notes
```

Comment:
This file is not generated right? Why do we need both this file and the file per release?

Reply:
It _is_ generated :-)